generate/main.go

```diff
@@ -281,6 +281,7 @@ func main() {
 					inline: []string{"opt_transaction"},
 					match:  []*regexp.Regexp{regexp.MustCompile("'COMMIT'|'END'")},
 				},
+				{name: "cancel_job", stmt: "cancel_job_stmt", replace: map[string]string{"a_expr": "job_id"}},
```

Comment:
Replacing `a_expr` with `job_id` means that the link goes nowhere. (Don't worry about this if there's no easy fix.)